ContactsApp/Exception.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Exception.swift
+//  ContactsApp
+//
+//  Created by Rohit Bansal on 04/10/15.
+//
+//
+
+import Foundation
+
+class Exception : NSObject, ErrorType {
```

Comment:
I would still like to prefer creating my custom ErrorType hierarchy and use them in validations, generic error messages etc. Always proved useful to me. Will fix the naming though! Thanks.